src/Shell.c

```diff
@@ -319,6 +372,8 @@ struct _Shell_Self
 
     PwrshPluginWkr_Ptrs managedPointers;
 
+    char *home;
```

Comment:
@paulcallen, there are a couple ways to go about this. I'm allocating on the heap; but this could just be an array of `PAL_MAX_PATH_SIZE`, which would eliminate needing to check and free this later.